vc-got.el

```diff
@@ -568,5 +568,64 @@ LIMIT limits the number of commits, optionally starting at START-REVISION."
                  (vc-got--diff file)))
               (t (error "Not implemented")))))))
 
+(defun vc-got-annotate-command (file buf &optional rev)
+  "Show annotated contents of FILE in buffer BUF. If given, use revision REV."
+  (let (process-file-side-effects)
+    (with-current-buffer buf
```

Comment:
For the time being, we can make VC happy by

```elisp
;; FIXME: vc-ensure-vc-buffer won't recognise this buffer as managed
;; by got unless vc-parent-buffer points to a buffer managed by got.
;; investigate why this is needed.
(set (make-local-variable 'vc-parent-buffer) (find-file-noselect file))
```

and with this `vc-got-previous-revision` should work.